src/server/services/helpers/setFileIndex.js

```diff
@@ -10,7 +10,7 @@ module.exports = function downloadIndex() {
     try {
         indexObject = JSON.parse(indexData);
     } catch (error) {
-        console.error("Sorry, i can`t parse to JSON indexDATA file. \n" + error);
+        throw new Error (`Sorry, i can not parse to JSON indexDATA file. \n ${error}`)
```

Comment:
Extra space, semicolon is required
throw new Error(`Sorry, i can not parse to JSON indexDATA file. \n${error}`);